VERSION "1.2"


CM_ "TM-630 for Truma interface and Onan Generator
Author: Damiano Liviero
for Winnebago Industries Inc.
";


NS_ :		NS_DESC_
		CM_
		BA_DEF_
		BA_
		VAL_
		CAT_DEF_
		CAT_
		FILTER
		BA_DEF_DEF_
		EV_DATA_
		ENVVAR_DATA_
		SGTYPE_
		SGTYPE_VAL_
		BA_DEF_SGTYPE_
		BA_SGTYPE_
		SIG_TYPE_REF_
		VAL_TABLE_
		SIG_GROUP_
		SIG_VALTYPE_
		SIGTYPE_VALTYPE_
		BO_TX_BU_
		BA_DEF_REL_
		BA_REL_
		BA_DEF_DEF_REL_
		BU_SG_REL_
		BU_EV_REL_
		BU_BO_REL_
		SG_MUL_VAL_

BS_:
BU_:

CM_ "EF64";
BO_ 2163172352 PROP_TM620_CONFIG_COMMAND: 8  Vector__XXX
    SG_ Operation:        0|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Unused:        8|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Water_Heater_Instance:        16|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Furnace_Instance:        24|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Truma_Device_Type:        32|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Thermostat_Instance:        40|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ LIN_Polling_Interval:        48|8@1+  (1,0)   [0|255] ""    Vector__XXX


VAL_ 2163172352 Operation
    50 "always use"
    255 "Data Invalid";

VAL_ 2163172352 Unused
    255 "Data Invalid";

VAL_ 2163172352 Water_Heater_Instance
    255 "Data Invalid";

VAL_ 2163172352 Furnace_Instance
    255 "Data Invalid";

VAL_ 2163172352 Truma_Device_Type
    0 "AquaGo/VarioHeat"
    1 "Combi"
    255 "Data Invalid";

VAL_ 2163172352 Thermostat_Instance
    255 "Data Invalid";

VAL_ 2163172352 LIN_Polling_Interval
    1 "100ms"
    2 "200ms"
    3 "300ms"
    4 "400ms"
    5 "500ms"
    255 "Data Invalid";


CM_ "EF96";
BO_ 2163185152 PROP_TM630_CONFIG_COMMAND: 8  Vector__XXX
    SG_ Operation:        0|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Unused:        8|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Water_Heater_Instance:        16|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Furnace_Instance:        24|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Truma_Device_Type:        32|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Thermostat_Instance:        40|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ LIN_Polling_Interval:        48|8@1+  (1,0)   [0|255] ""    Vector__XXX


VAL_ 2163185152 Operation
    50 "always use"
    255 "Data Invalid";

VAL_ 2163185152 Unused
    255 "Data Invalid";

VAL_ 2163185152 Water_Heater_Instance
    255 "Data Invalid";

VAL_ 2163185152 Furnace_Instance
    255 "Data Invalid";

VAL_ 2163185152 Truma_Device_Type
    0 "AquaGo/VarioHeat"
    1 "Combi"
    255 "Data Invalid";

VAL_ 2163185152 Thermostat_Instance
    255 "Data Invalid";

VAL_ 2163185152 LIN_Polling_Interval
    1 "100ms"
    2 "200ms"
    3 "300ms"
    4 "400ms"
    5 "500ms"
    255 "Data Invalid";


CM_ "EF44";
BO_ 2163164160 PROP_TM620_CONFIG_STATUS: 8  Vector__XXX
    SG_ Operation:
        0|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Unused:
        8|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Water_Heater_Instance:
        16|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Furnace_Instance:
        24|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Truma_Device_Type:
        32|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ Thermostat_Instance:
        40|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ LIN_Polling_Interval:
        48|8@1+  (1,0)   [0|255] ""    Vector__XXX
    SG_ LIN_Bus_Connection:
        56|2@1+  (1,0)   [0|3] ""    Vector__XXX
    SG_ Truma_Control_Panel_Status:
        58|2@1+  (1,0)   [0|3] ""    Vector__XXX
    SG_ Truma_Water_Heater_Error:
        60|2@1+  (1,0)   [0|3] ""    Vector__XXX
    SG_ Truma_Furnace_Error:
        62|2@1+  (1,0)   [0|3] ""    Vector__XXX

VAL_ 2163164160 Operation
    51 "Always"
    255 "Data Invalid";

VAL_ 2163164160 Unused
    255 "Data Invalid";

VAL_ 2163164160 Water_Heater_Instance
    255 "Data Invalid";

VAL_ 2163164160 Furnace_Instance
    255 "Data Invalid";

VAL_ 2163164160 Truma_Device_Type
    0 "AquaGo/VarioHeat"
    1 "Combi"
    255 "Data Invalid";

VAL_ 2163164160 Thermostat_Instance
    255 "Data Invalid";

VAL_ 2163164160 LIN_Polling_Interval
    1 "100ms"
    2 "200ms"
    3 "300ms"
    4 "400ms"
    5 "500ms"
    255 "Data Invalid";

VAL_ 2163164160 LIN_Bus_Connection
    0 "No"
    1 "Connected"
    3 "Data Invalid";

VAL_ 2163164160 Truma_Control_Panel_Status
    0 "Not Busy"
    1 "Busy"
    3 "Data Invalid";

VAL_ 2163164160 Truma_Water_Heater_Error
    0 "Ok"
    1 "Error"
    3 "Data Invalid";

VAL_ 2163164160 Truma_Furnace_Error
    0 "Ok"
    1 "Error"
    3 "Data Invalid";